client/src/router/index.js

```diff
@@ -113,6 +113,20 @@ const routes = [
       },
     },
   },
+  {
+    path: '/tools',
+    name: 'tools',
+    component: () =>
+      import(/* webpackChunkName: "games" */ '../views/Tools'),
```

Comment:
webpackChunkName should be unique, then you can tell what it is in the Network tab